backend/python/app/services/implementations/story_service.py

```diff
@@ -57,6 +57,14 @@ def create_story(self, story, content):
 
         return new_story
 
+    def get_stories_available_for_translation(self, language, level):
+        stories = set(
```

Comment:
Do we need a `set` here? `Story.query` should never return duplicate stories unless some weird joining goes on